.circleci/config.yml

```diff
@@ -153,7 +153,7 @@ commands:
           name: Run all tests
           command: |
             bash -l -c "python3 -m RLTest --version"
-            bash -l -c "make pytest"
+            bash -l -c "make test"
```

Comment:
You can state `shell: /bin/bash -l -eo pipefail` (parallel to `command:`) and avoid repeating the `bash -l -c` thing.

Reply:
I see this how it's being used through out the scripts...

Reply:
@gkorland The reason for `bash -l` is that it enables execution of scripts that were put in `/etc/profile.d` by various setup tools. CircleCI makes it possible to change the way bash is invoked for the whole `run` scriptlet, hence enabling `bash -l` for all commands at once.